Pipfile

```diff
@@ -24,6 +24,7 @@ sphinx-multiversion = {editable = true,git = "https://github.com/dls-controls/sp
 scanspec = {editable = true, extras = ["plotting", "graphql"], path = "."}
 # Force pipenv to install typing-extensions always (otherwise it skips on 3.8+)
 typing-extensions = "*"
+ipykernel = "*"
```

Comment:
What is this needed for?

Reply:
Looks like it was accidentally imported when using jupyter notebook, have since removed this and updated the pipfile.lock.